source/class/qx/tool/cli/Cli.js

```diff
@@ -549,6 +549,9 @@ Versions: @qooxdoo/compiler    v${qx.tool.compiler.Version.VERSION}
      * @return {Boolean}
      */
     isExplicitArg(option) {
+      /**
+       * @param option
+       */
```

Comment:
do we have to enable comments for mini helper functions like this?  They're normally pretty self explanatory, like this one